### PR TITLE
Fix typos: JS intermediate algorithm guide

### PR DESCRIPTION
**Pull request by husseyexplores**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

------------------
Fixes a minor typo. From `chack` to `check`

### CODE REVIEW

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md [hunk 1]
```diff
@@ -185,9 +185,9 @@ uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
 
 ### Code Explanation:
 
-*   We first use `concat()` with an empty array `<a href='http://exploringjs.com/es6/ch_maps-sets.html#_set' target='_blank' rel='nofollow'>]` as a starting point and the spread operator `...` to create an array out of the Arguments object and to flatten it at the same time
+*   We first use `concat()` with an empty array as a starting point and the spread operator `...` to create an array out of the Arguments object and to flatten it at the same time
 *   then we use the new ES2015 **Set** object to store only unique values
-*   (to learn more about Sets, read [here</a>)
+*   (To learn more about **Set**, [click here](http://exploringjs.com/es6/ch_maps-sets.html#sec_set)).
```
Comment (RandellDawson) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md]: As the Relevant Links section already has a link to about `Sets`, I suggest just to remove line 190 all together.  Everything else looks fine.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@husseyexplores See my suggestion for this PR.  Thank you for creating this PR.

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md (hunk 1 above)
Comment (husseyexplores) [reply guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md]: Thanks for the feedback. I removed the line 190.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@husseyexplores Changes look good.  We are reviewing an earlier PR (#34796).  Once a decision has been made on it, then we will merge this one.  Merging the other could create merge conflicts that may need to be dealt with, but we will resolve for you.

**Comment by RandellDawson** [comment]
Congrats @husseyexplores on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**